docs/UserGuide.md

```diff
@@ -75,78 +83,70 @@ Format: `help`
 
 ### Adding a person: `add`
 
-Adds a person to the address book.
+Adds a client to ClientBook.
 
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​`
+Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [i/POLICY_ID] [t/TAG]…​`
 
 <div markdown="span" class="alert alert-primary">:bulb: **Tip:**
 A person can have any number of tags (including 0)
 </div>
 
 Examples:
-* `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01`
-* `add n/Betsy Crowe t/friend e/betsycrowe@example.com a/Newgate Prison p/1234567 t/criminal`
+* (example with policy)`add n/John Doe p/98765432 e/johnd@example.com a/Ochard i/Policy_12738176 t/basic`
+* (example with no policy)`add n/Betsy Crowe e/betsycrowe@example.com a/Kent Ridge t/premium`
```

Comment:
Do you think a person should have a tag describing a policy if s/he has no policy number?